coders/qoi.c

```diff
@@ -77,7 +77,12 @@
 #define QOI_COLOR_HASH(C) (C.rgba.r*3 + C.rgba.g*5 + C.rgba.b*7 + C.rgba.a*11)
 
 /*
-  Typedef declaractions.
+ Static Const declarations.
+*/
+static const unsigned char qoi_endmarker[8] = {0,0,0,0,0,0,0,1};
```

Comment:
Could we not include this in the method itself and write all the bytes at one?

Reply:
using WriteBlobLongLong ?

Reply:
Probably easier with `WriteBlob` instead.

Reply:
a `WriteBlobMSBLongLong(image, 0x1ull);` would have done the trick :) but there is no such  function I'll use WriteBlob instead